remote/supervisord.xpra.conf

```
@@ -2,7 +2,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 [program:xpra]
-command=xpra start :10 --start=/home/techuser/.config/openbox/autostart --start-env=GTK_IM_MODULE=ibus --attach=yes --daemon=no --bind-tcp=0.0.0.0:10001 --min-quality=70
+command=xpra start-desktop :10 --start=/usr/local/bin/openbox-launcher.sh --attach=yes --daemon=no --bind-tcp=0.0.0.0:10001 --min-quality=70
```

```suggestion
command=xpra start-desktop :10 --start=/usr/local/bin/openbox-launcher.sh --resize-display="1920x1080" --attach=yes --daemon=no --bind-tcp=0.0.0.0:10001 --min-quality=70
```

I haven't tried it, but it might fix the splash screen issue. At least it says in xpra stdout: 

```
2024-10-16 13:42:21,481  desktop size is 8192x4096:
```

---

`--resize-display` seems to limit the maximum size of the desktop, instead of only affecting the pre-connection phase.

Fixed instead by moving the splash to the top left corner via Openbox window rules.